src/Explorer/Panes/AddDatabasePane.html

```diff
@@ -75,7 +75,7 @@
           <input id="database-id" type="text" aria-required="true" autocomplete="off" pattern="[^/?#\\]*[^/?# \\]"
               title="May not end with space nor contain characters '\' '/' '#' '?'"
               size="40" class="collid" data-bind="textInput: databaseId, hasFocus: firstFieldHasFocus, attr: { placeholder: databaseIdPlaceHolder }"
-              aria-label="Database id" autofocus>
+              aria-label="Keyspace id" autofocus>
```

Comment:
"Keyspace" only applies to Cassandra accounts so you need to change this to ensure it stills says "Database" for other API types

Reply:
Ok, I see, make sure only the cassandra accounts will change this aria-label